wp-content/themes/core/footer.php

```diff
@@ -1,7 +1,3 @@
 <?php declare(strict_types=1);
 
-use \Tribe\Project\Templates\Components\routes\page\Page_Controller;
-
-$c = Page_Controller::factory();
-
-$c->render_footer();
+get_template_part( 'components/document/footer/footer' );
```

Comment:
I think this should be fine, but I _really_ thought there was a reason we had to setup header/footer/sidebar this way (something about getting the data all setup in the controller first?)

Reply:
I suppose if you wanted to pass data to the header and or footer from the main controller, then it doesn't work as smooth. But it also breaks the WP template loading standard, which I think we should avoid changing if we can for better plugin support.